app/assets/stylesheets/content/_timelines.scss

```diff
@@ -71,6 +71,10 @@
   }
 }
 
+.controller-timelines #content {
+  min-height: 700px;
```

Comment:
This is quite a specific **hack** to allow the select2 menu to be visible under all circumstances.

You really should (I don't want to say "must", but you actually must) add a comment that clearly states the intent of this specific CSS rule.

Something like:

````CSS
/*HACK: Use a min-height to ensure that the timeline_select select2 is not cropped at the bottom*/
````